web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -76,6 +77,7 @@ export namespace MenuOverlay {
     @property({ type: Boolean, attribute: "show-arrow" }) showArrow = false;
     @property({ type: Boolean }) disabled = false;
     @property({ type: String }) placement: MenuOverlay.Placement = "bottom";
+    @property({ type: Boolean, attribute: "allow-hover-toggle" }) allowHoverToggle = false;
```

Comment:
I feel the name of the attribute is not conveying the purpose "allow-hover-toggle".
can we name as "triggeredByHover"?

Reply:
triggeredByHover looks a internal component level event. here this is kind of allowing or not flag. which looks fine to me.